opencdx-health/src/main/java/cdx/opencdx/health/controller/OpenCDXAnalysisEngineGrpcController.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2024 Safe Health Systems, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package cdx.opencdx.health.controller;
+
+import cdx.opencdx.grpc.service.health.*;
+import cdx.opencdx.health.service.OpenCDXAnalysisEngineService;
+import io.grpc.stub.StreamObserver;
+import io.micrometer.observation.annotation.Observed;
+import lombok.extern.slf4j.Slf4j;
+import org.lognet.springboot.grpc.GRpcService;
+import org.springframework.beans.factory.annotation.Autowired;
+
+/**
+ * gRPC Controller for AnalysisEngine
+ */
+@Slf4j
+@GRpcService
+@Observed(name = "opencdx")
+public class OpenCDXAnalysisEngineGrpcController extends AnalysisEngineServiceGrpc.AnalysisEngineServiceImplBase {
+    private final OpenCDXAnalysisEngineService openCDXAnalysisEngineService;
+
+    /**
+     * Constructor using the OpenCDXAnalysisEngineService
+     * @param openCDXAnalysisEngineService service to use for processing
+     */
+    @Autowired
```

Comment:
Missing  @Secured({})

Reply:
Added this attribute on all the grpc controller operations.
@Ghost-Programmer  -  Should this be added on the Rest controller operations as well?

Also this attribute is missing currently in majority of the existing controller classes, may be we need to address them individually.